specs/impact-yaml-suggestion.yaml

```diff
@@ -0,0 +1,66 @@
+$schema: "./impact-yaml-schema.json"
+name: app-name # The name of the root node (the applicatio as a whole)
+components: # The nodes under this root node
+  - name: nodename # The name of this node
+    tags: # common tags for identifying the resource
```

Comment:
I like the addition of tags

Reply:
Need to explore the schema for that. The array syntax is easily parsable whereas parsing the dict could be a complicated scenario.